client/app/scripts/superdesk-desks/views/main.html

```diff
@@ -3,12 +3,12 @@
     	<ul>
 	    	<li ng-repeat="v in views" ng-class="{active: v === view}">
 	    		<a href="" id="{{v}}" ng-click="setView(v)" translate>{{v}}</a>
-	    		<div ng-if="v === view && view === 'users'">
-	    			<input type="checkbox" ng-model="online_users" name="online_users" id="online_users" ng-change="changeOnlineUsers(online_users)">
-	    			<label translate>online users</label>
-	    		</div> 
+                        <div class="online-users" ng-if="v === view && view === 'users'">
```

Comment:
Oh, OK, then it's fine of course, I thought it was just about the `v`. Thanks for clarifying!